src/inmanta/main.py

```diff
@@ -210,10 +211,25 @@ def project_show(client: Client, project: str) -> None:
 @click.option("--name", "-n", help="The name of the new project", required=True)
 @click.pass_obj
 def project_create(client: Client, name: str) -> None:
+    create_project(client, name)
+
+
+def create_project(client: Client, name: str) -> None:
     project = client.get_dict("create_project", "project", {"name": name})
     print_table(["Name", "Value"], [["ID", project["id"]], ["Name", project["name"]]])
 
 
+@project.command(name="generate")
+@click.option("--name", "-n", help="The name of the new project", required=True)
+@click.option("--output-dir", "-o", help="Output directory path")
+@click.pass_obj
+def project_generate(client: Client, name: str, output_dir: Optional[str]) -> None:
+    cookiecutter(
+        "https://github.com/inmanta/inmanta-project-template.git", output_dir=output_dir, extra_context={"project_name": name}
+    )
+    create_project(client, name)
```

Comment:
In my opinion this command should only create the file structure of an Inmanta project. The project shouldn't be exported to an Inmanta server as well. Or the project creation on the server should at least be optional.